src/app/file/file-list/file-list.component.ts

```diff
@@ -206,6 +206,10 @@ export class FileListComponent implements OnInit, OnDestroy {
     }
 
     private removeFile(fileName: string): void {
+
+        if (!confirm('Do you really want to delete ' + fileName + '?')) {
+            return;
+        }
```

Comment:
I'd use the modal service here:
```
private removeFile(fileName: string): void {
        this.confirm(`Do you want to delete "${fileName}"?`, 'Delete a file', 'Delete')
            .pipe(switchMap(it => this.fileService.removeFile(this.path.absolutePath(fileName))))
            .takeUntil(this.ngUnsubscribe)
            .subscribe(it => this.loadData());
    }

    private confirm(text: string, title: string, confirmLabel: string): Observable<boolean> {
        const subj = new Subject<boolean>();
        this.modalService.show(ConfirmDialogComponent,
            {
                initialState: {
                    headerTitle: title,
                    confirmLabel: confirmLabel,
                    body: text,
                    isDiscardCancel: false,
                    callback: (value: boolean) => subj.next(value)
                }
            });
        return subj.asObservable().take(1).filter(v => v === true);
    }
``` 

it's better to move private confirm method into the separate service (like ModalService) and not copy past it everywhere. It's already in a few places... like in the submission-list.component.ts